tests/densify/test_mvs_patchmatchnet.py

```diff
@@ -0,0 +1,120 @@
+"""Unit tests for using Patchmatchnet method for dense point cloud reconstruction
+
+In this unit test, we make a simple scenario that eight cameras are around a circle (radius = 40.0).
+Every camera's pose is towards the center of the circle (0, 0, 0).
+
+There are 30 toy points uniformly located at the same line:
+  /  z - 2y = 0
+  \\ x = 0
+
+Authors: Ren Liu
+"""
+import unittest
+
+import numpy as np
+from gtsam import Cal3_S2, PinholeCameraCal3_S2, Point3
+from gtsam.examples import SFMdata
+
+from gtsfm.common.image import Image
+from gtsfm.common.gtsfm_data import GtsfmData, SfmTrack
+from gtsfm.densify.mvs_patchmatchnet import MVSPatchmatchNet
+
+# set the default image size as 800x600, with 3 channels
+DEFAULT_IMAGE_W = 400
+DEFAULT_IMAGE_H = 300
+DEFAULT_IMAGE_C = 3
+
+# set default track points, the coordinates are in the world frame
+DEFAULT_NUM_TRACKS = 30
+DEFAULT_TRACK_POINTS = [Point3(0, 0.5 * float(i), float(i)) for i in range(DEFAULT_NUM_TRACKS)]
+
+# set default camera intrinsics
+DEFAULT_CAMERA_INTRINSICS = Cal3_S2(
```

Comment:
Cal3Bundler is our default

Reply:
But `SFMdata.createPoses` in `gtsam/examples/SFMdata.py` requires `Cal3_S2`